### DISCUSSION TITLE
What does the thick arrow on a span's Service & Operation label mean?

### DISCUSSION THREAD

**Discussion by albertteoh** [jaegertracing/jaeger#3052] [Q&A] [answered]
... my guess would be something related to the `peer.service` tag, but I'm not sure.

For example:

<img width="660" alt="Screen Shot 2021-06-03 at 6 56 12 pm" src="https://user-images.githubusercontent.com/26584478/120623429-11116c80-c4a3-11eb-95a1-dbca170942b4.png">

**Comment by jpkrohling** [comment, marked as answer]
Looks like it's the "uninstrumented service" notation: https://github.com/jaegertracing/jaeger-ui/issues/594 / https://github.com/jaegertracing/jaeger-ui/pull/659

**Reply by albertteoh** [reply]
Great, thanks @jpkrohling !

**Comment by yurishkuro** [comment]
It represents a collapsed pair of client-server spans. It shows the names of both services as A->B.
